R/Analysis_box_heat.qmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Analysis Biomarker Boxplot and Phys. Att. Heatmap"
+format: html
+editor: visual
+---
+
+## Analysis: Biomarker levels for D/M status
+
+```{r}
+
+library('tidyverse')
+library('patchwork')
+library(ggplot2)
+library(scales)
+
+```
+
+```{r}
+
+getwd()
+```
+
+```{r}
+#load augmented data
+#| message: false
+diabetes_data <- read_tsv(file = "../data/04_diabetes_data_augment.tsv",
+                          na = "NA")
+
+```
+
+```{r}
+
+p1 <- ggplot(data = diabetes_data, aes(x = factor(dm_status), y = albumin, fill = factor(dm_status))) +
+  geom_boxplot() +
+  labs(
+    fill = "Status",
+    x = "Status",
+    y = "Albumin(g/dL)"
+  ) + 
+  scale_x_discrete(labels = c('Undiagnosed','Diagnosed Med','Diagnosed Unmed')) +
+  theme_minimal() +
+  theme(legend.position="none",
+        axis.text.x = element_text(angle = 15,
+        ))
+
+```
+
+```{r}
+p2 <- ggplot(data = diabetes_data, aes(x = factor(dm_status), y = bun, fill = factor(dm_status))) +
+  geom_boxplot() +
+  labs(
+    fill = "Status",
+    x = "Status",
+    y = "BUN (mg/dL)"
+  ) + 
+  scale_x_discrete(labels = c('Undiagnosed','Diagnosed Med','Diagnosed Unmed')) +
+  theme_minimal() +
+  theme(legend.position="none",
+        axis.text.x = element_text(angle = 15,
+        ))
+
+```
+
+```{r}
+
+p3 <- ggplot(data = diabetes_data, 
+             aes(x = factor(dm_status), 
+                                       y = creatinine, 
+                                       fill = factor(dm_status))) +
```

Comment:
Could we move y and fill more to the left?

Reply:
Data should be piped into the plot